executor/builder.go

```diff
@@ -552,15 +552,19 @@ func (b *executorBuilder) buildInsert(v *plannercore.Insert) Executor {
 	baseExec.initCap = chunk.ZeroCapacity
 
 	ivs := &InsertValues{
-		baseExecutor: baseExec,
-		Table:        v.Table,
-		Columns:      v.Columns,
-		Lists:        v.Lists,
-		SetList:      v.SetList,
-		GenColumns:   v.GenCols.Columns,
-		GenExprs:     v.GenCols.Exprs,
-		hasRefCols:   v.NeedFillDefaultValue,
-		SelectExec:   selectExec,
+		baseExecutor:  baseExec,
+		Table:         v.Table,
+		Columns:       v.Columns,
+		Lists:         v.Lists,
+		SetList:       v.SetList,
+		GenColumns:    v.GenCols.Columns,
+		GenExprs:      v.GenCols.Exprs,
+		hasRefCols:    v.NeedFillDefaultValue,
+		SelectExec:    selectExec,
+		rowCountLimit: uint64(b.ctx.GetSessionVars().DMLBatchSize / len(v.Table.Cols())),
```

Comment:
why divide `len(v.Table.Cols())`?

Reply:
It could balance the `rowCountLimit` with the table width.

Reply:
What if DMLBatchSize less than `len(v.Table.Cols())`?

Reply:
It will be set to `MinRowCountLimit`

Reply:
Fine, rowCounLimit is at least MinRowCountLimit